src/cargo/ops/cargo_compile.rs

```diff
@@ -1065,19 +1053,7 @@ fn generate_targets(
             // why this is done. However, if the package has its own
             // `package.target` key, then this gets used instead of
             // `$HOST`
```

Comment:
I think this comment, albeit somewhat reworded, would now belong on `Package::explicit_kinds`?